### DISCUSSION TITLE
是否有快捷获取EN服水月肉鸽对应关卡名和藏品译名的方法？

### DISCUSSION THREAD

**Discussion by Pitiedwzr** [MaaAssistantArknights/MaaAssistantArknights#4916] [Q&A] [answered]
目前正在做EN服水月肉鸽的适配，图像大部分都有了，关卡名和藏品的对应译名获取手打的话效率太低，而且指路麟这种难刷的更是基本看不到，是否有效率高一点的方法比如拆包之类的？

**Comment by MistEO** [comment]
关卡和藏品译名现在资源里就有吧，应该不用手动写的？

**Reply by MistEO** [reply]
好像是没更新，应该是资源更新器要适配下

**Reply by MistEO** [reply, marked as answer]
Sry, 这应该是资源更新器的一个 bug，只处理了傀影肉鸽。现在修复了

https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/4a919ea9501153f69a21805cadaf1933c6182c43
https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/21f465c4857d3ce3e469b805ed20757a6da8514f

**Reply by MistEO** [reply]
@178619 Hi, the replace names of levels and collections were not updated due to a bug in the resource updater. If you are adapting the updated replacement names, please pay attention to conflicts.

**Reply by 178619** [reply]
Thanks for notifying me and updating it, this will reduce required time for the work
